pkgs/by-name/li/liba2i/package.nix

```
@@ -0,0 +1,39 @@
+{ stdenv
+, fetchgit
+, lib
+, pkg-config
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "liba2i";
+  version = "unstable-at-2024-01-13";
+
+  src = fetchgit {
+    url = "git://www.alejandro-colomar.es/src/alx/alx/liba2i.git";
+    rev = "1d54d788038e671a5f85d79dd932b322247f03e2";
+    hash = "sha256-HqcQq8I91IYzBsFXoMdiivfc5iBCQcM/QmSw/jbuNhM=";
+  };
+
+  preConfigure = ''
```

```suggestion
  postPatch = ''
```

P.S.: IIRC some of those vars can be setted via `makeFlags`.

---

Cool, thanks!